.gitignore

```diff
@@ -24,4 +24,6 @@
 # Sublime
 *.sublime*
 
-/bin
\ No newline at end of file
+*gossamer*data*
```

Comment:
Please make this more specific. Let me know what pattern you are trying to ignore and I can probably suggest an alternative

Reply:
changed, was trying to ignore trie/gossamer_data